docs/source/index.rst

```diff
@@ -0,0 +1,21 @@
+Welcome to python-vibrato's documentation!
+==========================================
+
+`Vibrato`_ is a fast implementation of tokenization (or morphological analysis) based on the
+Viterbi algorithm. This is a Python wrapper for Vibrato.
+
+.. _Vibrato: https://github.com/daac-tools/vibrato
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   examples
+   api
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
+* :ref:`search`
```

Comment:
What are these references? Are they automatically created through Read the Docs?

Reply:
Those items are automatically generated by `sphinx-quickstart` command.
The built document is automatically pushed to https://python-vibrato.readthedocs.io/en/latest/

You can also build the documentation yourself:
```
$ cd ./python-vibrato  # root of this repo
$ python3 -m venv .env
$ source .env/bin/activate
$ pip install sphinx sphinx-rtd-theme
$ pip install .
$ cd docs
$ sphinx-build -M html ./source ./build  # build directory will be created
```

Reply:
@vbkaisetsu Thanks! I could compile it on my local. I confirmed that `genindex.html` and `search.html` were generated under `docs/build/html`; however, `py-modindex.html` could not found. Also on https://python-vibrato.readthedocs.io/en/latest/, `Module Index` seems to be 404. Is this expected?

Reply:
@kampersanda ping